pkg/northbound/gnmi/subscribe.go

```diff
@@ -299,6 +308,9 @@ func sendResponse(response *gnmi.SubscribeResponse, stream gnmi.GNMI_SubscribeSe
 func getChangeFromEvent(update events.ConfigEvent, mgr *manager.Manager) (string, *change.Change) {
 	target := events.Event(update).Subject()
 	changeID := update.ChangeID()
-	changeInternal := mgr.ChangeStore.Store[changeID]
+	changeInternal, ok := mgr.ChangeStore.Store[changeID]
+	if !ok {
+		log.Warning("No change found for: ", changeID)
```

Comment:
the change will be nil, and if nil we will get invalid pointer. should we check where this is called that if change == nil we skip the event ?

Reply:
We do - see line 176 above

Reply:
my bad